src/web/mjs/connectors/templates/MojoPortalComic.mjs

```diff
@@ -95,6 +95,7 @@ export default class MojoPortalComic extends Connector {
     async _handleConnectorURI(payload) {
         let request = new Request(payload.url, this.requestOptions);
         request.headers.set('x-referer', payload.referer);
+        request.headers.set('accept', 'image/webp,image/apng,image/*,*/*');
```

Comment:
Maybe it would be more appropriate placing this hack somewhere in [Request.mjs](https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/engine/Request.mjs#L440) with a condition to check for `uri.hostname.includes('imgur.com')` so it will apply to all imgur image links.

Reply:
its not just imgur though

Reply:
its literally every image on all of the sites changed

Reply:
see https://github.com/manga-download/hakuneko/issues/2925#issuecomment-760201931
hn gets the html one